ego/cli/sign.go

```diff
@@ -92,6 +90,10 @@ func (c *Cli) signWithJSON(conf *config.Config) error {
 }
 
 func (c *Cli) signExecutable(path string) error {
+	if err := c.checkUnsupportedImports(path); err != nil {
```

Comment:
we should do this in signWithJSON (and use conf.Exe as path) to cover all flavors of the sign command.